index.js

```diff
@@ -90,12 +90,12 @@ function invokeCommandError(command, isInternalCommand) {
   }
 
   const cwd = process.cwd();
-  logger.error(`No modules found for ${command}`);
+  logger.error(`No modules were found that contain the '${command}' command. Please check your syntax. For more information, use the 'help' command.`);
```

Comment:
Can we use "handle" instead of "contain" or perhaps another word you might like?

Reply:
I like "handle." ... I'll update throughout.